specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2017-12-01-preview/postgresql.json

```diff
@@ -0,0 +1,1725 @@
+    },
+    "OperationDisplay": {
+      "properties": {
+        "provider": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Operation resource provider name."
+        },
+        "resource": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource on which the operation is performed."
+        },
+        "operation": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Localized friendly name for the operation."
+        },
+        "description": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Operation description."
+        }
+      },
+      "description": "Display metadata associated with the operation."
+    },
+    "Operation": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the operation being performed on this particular object."
+        },
+        "display": {
+          "$ref": "#/definitions/OperationDisplay",
+          "readOnly": true,
+          "description": "The localized display information for this particular operation or action."
+        },
+        "origin": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The intended executor of the operation.",
+          "enum": [
+            "NotSpecified",
+            "user",
+            "system"
+          ],
+          "x-ms-enum": {
+            "name": "OperationOrigin",
+            "modelAsString": true
+          }
+        },
+        "properties": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "object"
+          },
+          "readOnly": true,
+          "x-ms-client-flatten": false,
+          "description": "Additional descriptions for the operation."
+        }
+      },
+      "description": "REST API operation definition."
+    },
+    "OperationListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Operation"
+          },
+          "description": "The list of resource provider operations."
+        }
+      },
+      "description": "A list of resource provider operations."
+    },
+    "LogFileProperties": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Log file name."
+        },
+        "sizeInKB": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Size of the log file."
+        },
+        "createdTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Creation timestamp of the log file."
+        },
+        "lastModifiedTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Last modified timestamp of the log file."
+        },
+        "type": {
```

Comment:
please rename this to fileType. What are the allowed values here?

Reply:
Will update API post GA